internal/controller/integrationpipeline/integrationpipeline_controller.go

```diff
@@ -118,7 +118,6 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	return controller.ReconcileHandler([]controller.Operation{
 		adapter.EnsureStatusReportedInSnapshot,
-		adapter.EnsureEphemeralEnvironmentsCleanedUp,
```

Comment:
one more reference needs to be removed from line 127.

Reply:
updated [here](https://github.com/konflux-ci/integration-service/pull/705/files#diff-28de961167fb89b7092c5a0acfe2f9f4543d218cd962d034fa9b60a86302b58bL128)